trogon/run_command.py

```diff
@@ -202,17 +208,14 @@ def _to_cli_args(self):
 
             # If the user has supplied any non-default values, include them...
             if values_supplied and not values_are_defaults:
-                for value_data in values:
+                for i, value_data in enumerate(values):
                     if not all(value == ValueNotSupplied() for value in value_data):
-                        args.append(option_name)
+                        # without multi-value (default): -u x -u y -u z
+                        # with multi-value: -u x y z
+                        if i == 0 or not option.option_schema.multi_value:
```

Comment:
If `multiple`, this arg accepts multiple values and the resulting CLI command will look like this: `-x 1 -x 2 -x 3`

If `multi_value`, then `multiple` is inherently True, and the resulting CLI command will look like: `-x 1 2 3`